packages/turbo-types/src/types/config.ts

```diff
@@ -3,6 +3,15 @@ export interface Schema {
   /** @default https://turbo.build/schema.json */
   $schema?: string;
 
+  /**
+   * A version of turbo that should be used with this project
+   *
+   * If turbo is invoked in a project with turboVersion specified, turbo will make
+   * sure to use the correct version. Only semver versions that include patch
+   * numbers are allowed.
+   */
+  turboVersion?: string;
```

Comment:
Name bikeshed: could it just be "version"? Or is that too ambiguous

Reply:
I think being explicit is cheap here. To me `version` could seem like a version of either the `turbo.json` schema itself or the monorepo itself. Not a strong opinion though.